docs/cep/tutorials/create-stream-app.md

```diff
@@ -209,12 +211,17 @@ Query Parameters:
 ### C8DB
 
 Syntax:
-
-	CREATE STORE StoreName WITH (type="database", collection="STRING", replication.type="STRING", collection.type="STRING", map.type='type', from="STRING", to="STRING") (strings);
-
+```sql
+   CREATE STORE StoreName WITH (type="database", collection="STRING", replication.type="STRING", collection.type="STRING", map.type='type', from="STRING", to="STRING") (strings);
+```
 Example:
-
-	CREATE STORE SweetProductionCollection WITH (type="database", collection="SweetProductionCollection", replication.type="local", map.type='json') (strings);
+```sql
+   CREATE STORE SweetProductionCollection WITH (type="database", collection="SweetProductionCollection", replication.type="local", collection.type="DOC", map.type='json') (name string, amount double);
+```
+Or equivalent using TABLE:
+```sql
+   CREATE TABLE GLOBAL SweetProductionCollection(name string, amount double);
+```
 
 Stream applications will use the c8db with the default query parameters explained in the chart below.
```

Comment:
Two things:
1. In all cases, replace "stream applications" with "stream workers." They used to call them stream applications, they don't anymore.
2. "C8DB" is also outdated terminology. Replace with "Macrometa" or the exact name of what is being accessed, like a collection.